README.md

```diff
@@ -1,6 +1,12 @@
 # Leaphy Micropython
 Source Code for the Leaphy MicroPython Library
 
+# Supported microcontrollers
+* Maker Nano RP2040
+* Pico W
+* Nano RP2040 Connect
+
+
 # How to install the package on your microcontroller
 First connect your microcontroller to the wifi, you need to do this in the REPL mode in a terminal
```

Comment:
This could use a bit more explaining, what's  repl mode and how do you get there (maybe mention thonny or tio on linux).

Reply:
Is goed